scripts/vmbuild.rb

```diff
@@ -39,7 +39,7 @@
 STORAGE_DIR         = BUILD_BASE.join("storage")
 ISOS_DIR            = Pathname.new("/build/isos") # NOTE: We are not using run-specific BUILD_BASE in order to share the source iso across builds
 IMAGES_DIR          = BUILD_BASE.join("images")
-ISO_FILE            = ISOS_DIR.glob("CentOS-Stream-8-x86_64-*-dvd1.iso").sort.last.expand_path
+ISO_FILE            = ISOS_DIR.glob("CentOS-Stream-9-*-x86_64-dvd1.iso").sort.last.expand_path
```

Comment:
I tried to quickly to change this to the much smaller boot ISO, but that didn't work.  I'll open an issue to investigate that.  It could shave 10-20 minutes per image off of the build time.

Reply:
https://github.com/ManageIQ/manageiq-appliance-build/issues/564